base/doc/clsguide.tex

```diff
@@ -807,90 +592,84 @@ \subsection{Example: a newsletter class}
    \setlength{\textwidth}{17.5cm}
    \setlength{\textheight}{25cm}
 \end{verbatim}
-In practice, a class would need more than this: it would provide
-commands for issue numbers, authors of articles, page styles and so
-on; but this skeleton gives a start.  The |ltnews| class file is not
-much more complex than this one.
+In practice, a class would need more than this: it would provide commands for
+issue numbers, authors of articles, page styles and so on; but this skeleton
+gives a start. The |ltnews| class file is not much more complex than this one.
```

Comment:
I didn't write this: it's copy-pasted.

Reply:
It may have been appropriate back then, my point is I doubt it is a good reference now and if you look into ltnews.cls I think it is not (and longer, or perhaps never was)